code/Friend.java

```diff
@@ -11,26 +11,26 @@ public Friend(String firstName, String lastName, String email) {
         this.lastName = lastName;
         this.email = email;
     }
-
-    public String getFirstName() {
-        return firstName;
-    }
-
-    public void setFirstName(String firstName) {
-        this.firstName = firstName;
-    }
-
-    public String getLastName() {
-        return lastName;
-    }
-
-    public void setLastName(String lastName) {
-        this.lastName = lastName;
-    }
-
-    public String getEmail() {
-        return email;
-    }
+//
+//    public String getFirstName() {
+//        return firstName;
+//    }
```

Comment:
I don't really follow the logic for this removal. The methods are unused in this example, but they still seem like something a `Friend` should have. Getters and setters are javabean standard (which this is _almost_ a bean) -- your data holding class isn't super useful if the only way to look at the data is the `toString` and `equals`.

Reply:
Added them back, especially with the addition of `get`, which returns a `Friend` that can then be played with.